geps/gep-957.md

```diff
@@ -3,6 +3,12 @@
 * Issue: [#957](https://github.com/kubernetes-sigs/gateway-api/issues/957)
 * Status: Experimental
 
+> **Note**: This GEP is exempt from the [Probationary Period][gepprob] rules of
+> our GEP overview as it existed before those rules did, and so it has been
+> explicitly grandfathered in.
+
+[gepprob]:https://gateway-api.sigs.k8s.io/geps/overview/#5-probationary-period
```

Comment:
Nit: These links are outdated now.

Reply:
Thanks, fixed in e11581a772a294b121db610a7390380d55de6379